frontend/public/components/utils/firehose.jsx

```diff
@@ -127,6 +127,7 @@ const stateToProps = ({k8s}, {resources}) => {
   };
 };
 
+/** @type any */
```

Comment:
This also has to do with typing of redux connected components. The `forceUpdate` prop being used by overview is causing a type error and I had difficulty figuring out how to fix it (especially since this is not a TS file). I didn't think refactoring this into typescript would be a good idea (for the scope of this story) since it is basically the nerve center of the app. @alecmerdler any suggestions? Here is the specific typescript error that we get without the JSDoc type declaration:

```sh
ERROR in /Users/jonjacks/code/tectonic/frontend/public/components/overview/index.tsx(750,41):
TS2339: Property 'forceUpdate' does not exist on type 'IntrinsicAttributes & IntrinsicClassAttributes<Component<Pick<{ k8sModels?: Map<string, any>; }, ...'.
```

Reply:
```jsx
<Firehose resources={resources} forceUpdate={true}>
  <OverviewDetails
    namespace={namespace}
    selectedItem={selectedItem}
    title={title}
  />
</Firehose>
```

Reply:
I'd try to update this comment from firehose.jsx to add `forceUpdate`

```jsx
  /** @augments {React.Component<{k8sModels?: Map<string, K8sKind}>} */
  class Firehose extends React.Component {
```